packages/tenant-process/src/utilities/errorMappers.ts

```diff
@@ -229,3 +229,13 @@ export const m2mRevokeCertifiedAttributeErrorMapper = (
     )
     .with("tenantIsNotACertifier", () => HTTP_STATUS_FORBIDDEN)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const assignTenantDelegatedProducerFeatureErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with(
+      "tenantAlreadyHasDelegatedProducerFeature",
+      () => HTTP_STATUS_CONFLICT
+    )
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
```

Comment:
Maybe mapping to 403 for the operationForbidden thrown by assertRequesterIPAOrigin is missing?

Reply:
Thanks Eric! This issue is solved in [another PR](https://github.com/pagopa/interop-be-monorepo/pull/1189)